types/kdbush/index.d.ts

```diff
@@ -0,0 +1,25 @@
+// Type definitions for kdbush 1.0
+// Project: https://github.com/mourner/kdbush
+// Definitions by: DenisCarriere <https://github.com/DenisCarriere>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+type Points = number[][];
```

Comment:
True, however 95% of the time some one will be using [`@types/geojson`](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/geojson) and the [Point types are defined as `number[]`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/geojson/index.d.ts#L20).